front/app/components/Form/Components/Controls/visibilityUtils.ts

```diff
@@ -174,19 +176,19 @@ export const isVisible = (
 
 // This returns the elements on a page that are visible based on the data and the other option selection. You can pass returnHidden as true to get the hidden elements
 export const extractElementsByOtherOptionLogic = (
-  page: PageType,
+  page: any,
```

Comment:
Although this works. There are different types being passed in that seem to be incompatible so I can't get it working without using any.